### PR TITLE
fix spacy example

### PR DESCRIPTION
**Pull request by RafalSkolasinski**
Closes https://github.com/SeldonIO/seldon-core/issues/1912.

Bumps its dependencies and adjust model downloading to deployments running nodes as non-root users.

### CODE REVIEW

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **rafalskolasinski**

If they are not already assigned, you can assign the PR to them by writing `/assign @rafalskolasinski` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=SeldonIO%2Fseldon-core).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by seldondev** [comment]
Thu Jun 11 14:50:46 UTC 2020
The logs for [pr-build] [1] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1935/1.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1935 --build=1

**Comment by seldondev** [comment]
Thu Jun 11 14:50:54 UTC 2020
The logs for [lint] [2] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1935/2.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1935 --build=2

File: examples/models/sklearn_spacy_text/sklearn_spacy_text_classifier_example.ipynb [hunk 1]
```diff
@@ -45,19 +45,85 @@
   },
```
Comment (axsaucedo) [inline examples/models/sklearn_spacy_text/sklearn_spacy_text_classifier_example.ipynb:45]: <p>Would be worth clearing the logs for long outputs like this one</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/seldon-core/pull/1935/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (axsaucedo) [inline examples/models/sklearn_spacy_text/sklearn_spacy_text_classifier_example.ipynb:45]: <p>Same with this one</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/seldon-core/pull/1935/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Comment (RafalSkolasinski) [reply examples/models/sklearn_spacy_text/sklearn_spacy_text_classifier_example.ipynb:45]: should be fixed now

Comment (RafalSkolasinski) [reply examples/models/sklearn_spacy_text/sklearn_spacy_text_classifier_example.ipynb:45]: same

**Comment by seldondev** [comment]
Thu Jun 11 15:02:20 UTC 2020
The logs for [pr-build] [3] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1935/3.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1935 --build=3

**Comment by seldondev** [comment]
Thu Jun 11 15:02:21 UTC 2020
The logs for [lint] [4] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1935/4.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1935 --build=4

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/SeldonIO/seldon-core/pull/1935#issuecomment-642802100" title="Approved">axsaucedo</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=SeldonIO%2Fseldon-core).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)~~ [axsaucedo]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by seldondev** [comment]
Thu Jun 11 16:49:00 UTC 2020
The logs for [pr-build] [5] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1935/5.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1935 --build=5

**Comment by seldondev** [comment]
Thu Jun 11 16:49:00 UTC 2020
The logs for [lint] [6] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1935/6.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-1935 --build=6

**Pull request merged**